README.md

```diff
@@ -44,20 +44,55 @@ func main() {
 }
 ```
 
+```go
+package main
+
+import (
+	kit "github.com/rarimo/zkverifier-kit"
+	"github.com/rarimo/zkverifier-kit/root"
+)
+
+func main() {
+	pollVerifier := root.NewProposalSMTVerifier(rpcURL, reqTimeout)
+	
+	v, err := kit.NewVerifier(
+		nil,
+		kit.WithProofType(kit.PollParticipation),
+		kit.WithVerificationKeyFile("key.json"),
+		kit.WithEventID("304358862882731539112827930982999386691702727710421481944329166126417129570"),
+	)
+	if err != nil {
+		// ...
+	}
+	// data is an abstract event data that you expect to be in proof
+	err = v.VerifyProof(proof,
+		kit.WithPollParticipationEventID(pollEventID) ,
+		kit.WithPollRootVerifier(v.WithContract(pollContract)))
+	if err != nil {
+		// ...
+	}
+}
+```
 Let's break this down.
 
 ### Configurable root verifier
 
 Firstly, you instantiate identity root verifier, which will verify the
 `IdStateRoot` public signal with contract call. You can refer to our
-generated contract bindings in [poseidonsmt](internal/poseidonsmt) package.
+generated contract bindings in [poseidonsmt](internal/poseidonsmt) and [proposalsmt](internal/proposalsmt) package.
 However, maybe, you would like to create the verifier from config map.
 
 Here is configuration sample that you should have in `config.yaml` of your app:
 ```yaml
-root_verifier:
-  rpc: https://your-rpc
-  contract: 0x...
+verifier:
```

Comment:
Add examples of using config. It might be either ProvideVerifier() with specified Type or custom initialization without Provider.